openstack/tools/instance_launch.sh

```diff
@@ -49,8 +50,14 @@ EOF
             shift
             old_IFS=${IFS}
             IFS=','
-            read -r -a net_names <<< "$1"
+            read -r -a new_net_names <<< "$1"
             IFS=${old_IFS}
+            if ((net_names_set == 0)); then
+                net_names=( ${new_net_names[@]} )
+                net_names_set=1
+            else
+                net_names=( ${net_names[@]} ${new_net_names[@]} )
+            fi
```

Comment:
Just curious if you could have used += here, for example:

net_names+=(" ${new_net_names[@]}")

Untested of course.

Reply:
Thanks, that should work :smile: